# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/inspector/InspectorFontFace.cpp
# Commit: 93b7e171c74a
# Full Hash: 93b7e171c74a6aa88d98979009eb0e299e2b61a3
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 1. Split FontFace(Set) into DOM facing and implementation classes. r=emilio
#   
#   This patch aims to split the implementation details of FontFace(Set)
#   into FontFace(Set)Impl classes. The Impl variants have threadsafe
#   refcounting and are intended to be the basis for worker support for
# ==============================================================================

diff -r c45dacd63805 -r 93b7e171c74a layout/inspector/InspectorFontFace.cpp
--- a/layout/inspector/InspectorFontFace.cpp	Tue Jun 28 14:12:34 2022 +0000
+++ b/layout/inspector/InspectorFontFace.cpp	Tue Jun 28 14:27:20 2022 +0000
@@ -72,7 +72,7 @@
       FontFaceSet::UserFontSet* fontSet =
           static_cast<FontFaceSet::UserFontSet*>(mFontGroup->GetUserFontSet());
       if (fontSet) {
-        FontFaceSet* fontFaceSet = fontSet->GetFontFaceSet();
+        FontFaceSetImpl* fontFaceSet = fontSet->GetFontFaceSet();
         if (fontFaceSet) {
           rule = fontFaceSet->FindRuleForEntry(mFontEntry);
         }